guide/creating-your-bot/creating-commands.md

```diff
@@ -81,45 +105,105 @@ const rest = new REST({ version: '9' }).setToken(token);
 :::
 ::::
 
-Once you fill in these values, run `node deploy-commands.js` in your project directory to register your commands to a single guild. It's also possible to [register commands globally](/interactions/registering-slash-commands.md#global-commands).
+Focus on these variables:
+
+- `clientId`: Your client's id
+- `guildId`: Your development server's id
+- `commands`: An array of commands to register. 
+
+The [`SlashCommandBuilder()`](/popular-topics/builders.md#Slash-command-builders) from the [`@discordjs/builders`](https://github.com/discordjs/builders) module is used to build the structure for your commands. As you can see from the deployment script above, the structure of the `/ping` command has a name `ping` and a description `Replies with pong!`. The structure can include other things such as subcommand, options, choices, permission, etc. You can refer to our [in-depth page on slash commands](/interactions/registering-slash-commands.md) to learn more about it.
 
 ::: tip
-You only need to run `node deploy-commands.js` once. You should only run it again if you add or edit existing commands.
+In order to get your client and guild IDs, open Discord and go to your settings. On the "Advanced" page, turn on "Developer Mode". This will enable a "Copy ID" button in the context menu when you right-click on a server icon, a user's profile, etc.
 :::
 
-## Replying to commands
+Once you fill in those variables, run:
+```sh:no-line-numbers
+node deploy-commands.js
+``` 
+in your project directory to register the structure of your slash commands to your specified guild.
+
+Congratulations! 🎉
 
-Once you've registered your commands, you can listen for interactions via <DocsLink path="class/Client?scrollTo=e-interactionCreate" /> in your `index.js` file.
+You've successfully registered your slash commands. Go ahead and type `/` in your guild and you should be able to see your slash commands already:
 
-You should first check if an interation is a command via <DocsLink path="class/Interaction?scrollTo=isCommand" type="method">`.isCommand()`</DocsLink>, and then check the <DocsLink path="class/CommandInteraction?scrollTo=commandName">`.commandName`</DocsLink> property to know which command it is. You can respond to interactions with <DocsLink path="class/CommandInteraction?scrollTo=reply">`.reply()`</DocsLink>.
+![Command Picker](./images/commandpicker.png)
+
+## Replying to slash commands
```

Comment:
Sorry but I don't agree with the changes made to this section 😦 What was previously straightforward now goes into detail in areas that it doesn't need to. Please revert this section